outputs.tf

```diff
@@ -1,8 +1,12 @@
-#output "ecs_cluster" {
-#  value = ["${aws_subnet.private.*.id}"]
-#}
-
 output "cluster_id" {
   value = "${aws_ecs_cluster.cluster.id}"
 }
-# ECS servers
+
+
+output "autoscaling_group" {
+  value = {
+    id   = "${aws_autoscaling_group.ecs.id}"
+    name = "${aws_autoscaling_group.ecs.name}"
+    asrn = "${aws_autoscaling_group.ecs.arn}"
```

Comment:
I see both perspectives, on one hand it's a slightly different behavior then we use in most cases, on the other, having a map could be useful, and so long as we've clearly documented it I think we're good.